src/core/datagram.h

```diff
@@ -37,7 +37,6 @@ typedef struct QUIC_DATAGRAM {
     // send.
     //
     BOOLEAN SendEnabled : 1;
-
```

Comment:
nit: undo this change

Reply:
Looks like you added some trailing spaces. 😄 Please delete those too. You can configure most code editors to do this automatically.

Reply:
LOL I'll install a vscode extension for that or smth